### DISCUSSION TITLE
字幕默认开启

### DISCUSSION THREAD

**Discussion by ddqlsep** [the1812/Bilibili-Evolved#3745] [功能建议 / Ideas]
希望能更加自动打开字幕的功能，经常性的忘记开启字幕，而且每次都要手动很麻烦。

**Comment by the1812** [comment]
我测试就是默认开启的啊

**Reply by Benature** [reply]
请问默认开启是安装了哪个插件/组建吗？我已经安装了「下载字幕」（搜字幕好像只搜到这个）

**Reply by the1812** [reply]
不需要装啥, b站默认就是开启的

**Reply by Benature** [reply]
我这里无论是刷新还是自动播放下一p都是默认关闭字幕的，需要手动点开。

在没有装插件的safari也试了，没有默认开启诶。

>比如 https://www.bilibili.com/video/BV1wq4y1M7qf?p=17

**Reply by the1812** [reply]
那这个呢? https://www.bilibili.com/video/BV1Gf4y1y7wc/

**Reply by Benature** [reply]
字幕自动开了，看来是不同视频有区别？🤦‍♂️

**Reply by the1812** [reply]
你那个是 AI 字幕, 并不是视频带有 CC 字幕

**Reply by Benature** [reply]
那可以支持所有字幕都默认开启吗？

**Reply by SaulJWu** [reply]
> 你那个是 AI 字幕, 并不是视频带有 CC 字幕

什么时候支持自动点开AI字幕

**Reply by the1812** [reply]
目前没有计划

**Reply by XMB-7** [reply]
请问目前有意向自动开启AI字幕么, 或者增设一个插件设置选项, 当视频无人工字幕时自动开启AI字幕. 
毕竟如今AI字幕已经非常普遍了, 我都养成了看视频三倍速主看AI字幕的习惯, 每次手动开启还是很麻烦的. 我现在是写了一个ahk脚本识别相应图标自动点击, 虽然也解决问题, 但还是希望能够自动开启, 每次都能少按一次键盘, 累积起来也是很省事了.

**Reply by Sefank** [reply]
同求自动开启AI字幕

**Comment by Ja-hard** [comment]
话说这个 feature 目前是个什么情况，有实现吗？或者可以安装的组件/插件？

**Reply by Geeyun-JY3** [reply]
看置顶：https://github.com/the1812/Bilibili-Evolved/discussions/1297#discussion-57881

> 如果是 2021-10-14 之后提出的功能请求类, 我会尝试使用 Tag 标明状态:
> 
> - ……
> - `feat: reviewed`: 已了解此建议 (但不一定是马上做, 我空闲时间有限, 只挑一些感兴趣的做)
> - ……
